sdcm/results_artifacts.html

```diff
@@ -0,0 +1,86 @@
+{% extends 'results_base.html' %}
+{% block body %}
+    <h3>Test: {{ test_name }}</h3>
+
+    <h3>Test-id: {{ test_id }}</h3>
+
+    <h3>Test details</h3>
+    <div>
+        <ul>
+            <li>Start time: {{ start_time }}</li>
+            <li>End time: {{ end_time }}</li>
+            <li>Build URL: <a href="{{ build_url }}">link</a></li>
+            <li>Test run by User: <b>{{ username }}</b></li>
+        </ul>
+    </div>
+
+    <h3>
+        <span>System under test </span>
+    </h3>
+    <div>
+        <ul>
+            <li>ScyllaDB version: {{ scylla_version }}</li>
+            <li>ScyllaDB repo: {{ scylla_repo }}</li>
+            <li>Actual GCE image used: {{ scylla_node_image }}</li>
+        </ul>
+    </div>
+
+    <h3>
+        <span>Test results </span>
+    </h3>
+    <div> {{ test_status[0] }} </div>
+    {% if test_status[1] %}
+        <div>
+            {% for line in test_status[1] %}
+                <pre>{{ line }}</pre>
+            {% endfor %}
+        </div>
+    {% endif %}
+
+    <h3>
+        <span>Events summary</span>
+    </h3>
+    <div>
+        <ul>
+            {% for k, v in events_summary.items() %}
+                <li>{{k}} : {{v}}</li>
+            {% endfor %}
+        </ul>
+    </div>
+
+    <h3>
+        Hydra commands:
+    </h3>
+    <div>
+        <ul>
+            <li>Restore Monitor Stack command: $ hydra investigate show-monitor {{ test_id }}</li>
```

Comment:
remove this, since we don't provision scylla-monitoring stack

Reply:
done